src/main/java/org/opensearch/neuralsearch/highlight/SemanticHighlighter.java

```diff
@@ -60,6 +60,11 @@ public HighlightField highlight(FieldHighlightContext fieldContext) {
             return null;
         }
 
+        String[] preTags = fieldContext.field.fieldOptions().preTags();
+        String[] postTags = fieldContext.field.fieldOptions().postTags();
+        // The pre- and post- tags are provided by the user or defaulted to <em> and </em>
+        semanticHighlighterEngine = semanticHighlighterEngine.withTags(preTags[0], postTags[0]);
```

Comment:
I think we should not modify the semanticHighlighterEngine which is not thread safe. Also, creating new semanticHighlighterEngine for every highlight request is not efficient. Instead we should just pass the tag to `semanticHighlighterEngine.getHighlightedSentences` to get correct response.

Reply:
Updated to just use method instead of new instance.